Dockerfile

```diff
@@ -54,10 +54,9 @@ RUN apt --quiet --yes update \
         jq \
         unzip \
         gettext \
-        texlive-latex-extra \
-        texlive-fonts-recommended \
         cm-super \
         postgresql-common \
+    && apt --quiet --yes install --no-install-recommends texlive-latex-extra \
```

Comment:
Question: Why is this on a separate line - The above already uses `--no-install-recommends`. Does it need to be installed in a separate transaction?

Reply:
You're right, these can be combined, addressed in [ad2bb93](https://github.com/ONSdigital/dis-wagtail/pull/208/commits/ad2bb9349c176f2257181c1050284e9687450740)